frontend/src/components/common/form/StepForm/context.tsx

```diff
@@ -116,7 +119,10 @@ export const StepperFormProvider: React.FC<IStepperFormProviderProps> = ({
   };
 
   const gotoStep = (index: number) => {
-    if (index > 0 && index < steps.length) {
+    if (!validateCurrentStep()) {
```

Comment:
handle validation by default now. This ended up being required as there was no validation on moving directly to steps. This allowed the user to leave a step in error and then continue, which isn't supported by our workflow.